src/platform_impl/linux/wayland/window/mod.rs

```diff
@@ -139,6 +144,16 @@ impl Window {
             )
             .map_err(|_| os_error!(OsError::WaylandMisc("failed to create window.")))?;
 
+        // Set CSD frame config
+        #[cfg(feature = "sctk-adwaita")]
+        if let Some(theme) = platform_attributes.csd_theme {
```

Comment:
Would you mind checking env variable `WINIT_WAYLAND_CSD_THEME` for `dark` or `light` when `None` was found?

You should also add it to the changelog and to the new `with_wayland_csd_theme` method saying that it could be used in case downstream application doesn't want to expose settings for wayland client side decorations to the user on its own.

Should be a minor change.

Reply:
Done